tests/Http/PostTest.php

```diff
@@ -615,12 +615,15 @@ public function testPostsIndexAsAdmin()
      */
     public function testPostsIndexAsAdminHiddenPosts()
     {
-        $this->markTestIncomplete();
+        // $this->markTestIncomplete();
         // Admins should see all posts.
         factory(Post::class, 'accepted', 10)->create();
 
         $hiddenPost = factory(Post::class, 'accepted')->create();
         // $hiddenPost->tag('Hide In Gallery');
+        $this->withAdminAccessToken()->postJson('api/v3/posts/' . $hiddenPost->id . '/tags', [
+            'tag_name' => 'Hide In Gallery',
+        ]);
```

Comment:
@DFurnes I don't think there is anything wrong with the Gateway test tokens but rather I think there is something that the test doesn't like about `->tag()` in line 623. When I tag a post via the API, the test passes! 

Also, when I comment out this line, the `tags` array is returned in the response (we were seeing this disappeared in the response as soon as we added the `$hiddenPost->tag('Hide In Gallery');` line).  

When tagging with the API, not only does the test pass but also the `tags` array is returned in the response as expected as well.

Reply:
@chloealee sorry to just bop in, but is there nothing in the logs about this failure?

Reply:
@sbsmith86 don't be sorry - thanks for this suggestion! The logs say that the post is successfully tagged: 
```
[2018-05-10 19:30:52] local.INFO: post_tagged {"id":11,"tag":"hide-in-gallery"} {"user_id":null,"client_id":null,"request_id":null}
```

Reply:
This is what the log says when I tag via the API: 
```
[2018-05-10 19:33:44] local.INFO: post_tagged {"id":11,"tag":"hide-in-gallery"} {"user_id":"5589c991a59dbfa93d8b45ae","client_id":"phpunit","request_id":null}
```
I don't think it should get mad by the `null`ed `user_id` and `client_id`?

Reply:
when the test runs and fails, the message you get isn't very descriptive, but sometimes I look in the logs to see more details. It might be nice to see what happens in the logs on the failure instead of the successful API request?

Reply:
oh interesting idea, thanks @sbsmith86 !